modules/lsm/lsm_local.c

```diff
@@ -132,6 +132,13 @@ get_blk_path (UDisksDriveLsmLocal *ud_drv_lsm_local,
 
   ud_blk = udisks_object_get_block (UDISKS_OBJECT (ud_lx_blk_obj));
   blk_path = udisks_block_dup_device (ud_blk);
+  if (blk_path == NULL)
+    {
+      g_dbus_method_invocation_return_error
+        (invocation, UDISKS_ERROR, UDISKS_ERROR_FAILED,
+         "Failed to retrieve block path of specified disk drive");
```

Comment:
Agreed with the 80 chars limit being rather obscure these days. Looking at other files the whole LSM module could use some codestyle cleanup. Trying to keep it out of scope of this PR for the moment though. 

I've fixed your concern so that the codestyle is inconsistent now that would hopefully push us to do the cleanup soon... :-)